src/lkm_config.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
We should probably add an explanatory `"_comment"` entry at the beginning of this file (like we do for some of the checks), so that it is easier for people to find out what this extra configuration file is for.